### PR TITLE
Apply import interval only when reasonable

### PR DESCRIPTION
**Pull request by mj-saunders**
## Description

When importing videos from another service, an interval can be applied between each download.  
It only really makes sense to apply this interval when the last attempted download actually happened, and not when it was skipped.

## Has this been tested?

- [ ] 👍 yes, I added tests to the test suite
- [x] 💭 no, because this PR is a draft and **may** still needs work
- [x] 🙅 no, because this PR does not update server code
- [ ] 🙋 no, because I need help 

## Note

I originally had to edit code in `dist/` directly, as I needed the functionality immediately.  
I have tested it only manually and it appears to behave as I would expect.

I am not an experienced `type/javascript` programmer and would like to confirm that the fix is as you might expect or want it to be.  
The problem is trivial, but is the solutions satisfactory?

### CODE REVIEW

File: server/tools/peertube-import-videos.ts [hunk 1]
```diff
@@ -184,6 +185,8 @@ async function processVideo (parameters: {
   } catch (err) {
     log.error(err.message)
   }
+
+  return 0;
```
Comment (Chocobozzz) [inline server/tools/peertube-import-videos.ts:189]: Use `true`/`false` instead

File: server/tools/peertube-import-videos.ts [hunk 2]
```diff
@@ -95,14 +95,15 @@ async function run (url: string, username: string, password: string) {
 
   log.info('Will download and upload %d videos.\n', infoArray.length)
 
+  let skipInterval = true;
```
Comment (Chocobozzz) [inline server/tools/peertube-import-videos.ts:98]: extra `;`

File: server/tools/peertube-import-videos.ts [hunk 3]
```diff
@@ -184,6 +185,8 @@ async function processVideo (parameters: {
   } catch (err) {
     log.error(err.message)
   }
+
+  return false;
```
Comment (Chocobozzz) [inline server/tools/peertube-import-videos.ts:189]: extra `;`

Comment (mj-saunders) [reply server/tools/peertube-import-videos.ts:189]: Well they were stupid mistakes.
Would it be acceptable to force push the change?
Or do a fresh commit, assuming perhaps that you would anyway squash merge later?

Comment (rigelk) [reply server/tools/peertube-import-videos.ts:189]: Both are usually acceptable.

**Comment by Chocobozzz** [comment]
Thanks!

**Pull request merged**